content/learning-paths/microcontrollers/asm/setup_mdk6.md

```diff
@@ -0,0 +1,105 @@
+---
+# User change
+title: "Setting up a Project in Keil Studio (VS Code)" 
+
+weight: 4 # 1 is first, 2 is second, etc.
+
+# Do not modify these elements
+layout: "learningpathall"
+---
+{{% notice  Note%}}
+If using Keil μVision, go to [Setting up a Project in Keil MDK (μVision)](..//setup_mdk5/).
+{{% /notice %}}
+
+## Create a new project (csolution)
+
+Keil Studio projects are based on the [CMSIS Solution](https://github.com/Open-CMSIS-Pack/cmsis-toolbox/blob/main/docs/YML-Input-Format.md) standard.
+
+Open the `VS Code` IDE, and select `File` > `New File` from the `File` menu. (`Ctrl`+`Alt`+`Windows`+`N`). You will be prompted for the type of file. Select `New Solution` (`Arm CMSIS Solution`).
+
+The `Create New Solution` window will open. Click the `Target Device` pulldown, and search for `ARMCM4`.
+
+From the `Templates, Reference Applications, and Examples` pulldown, select `Blank Solution`.
+
+Ensure `Arm Compiler 6` is the selected compiler.
+
+Enter an appropriate `Solution Name`. This will define the folder name that the project will be created into. You can also change the folder location if necessary.
+
+Click `Create`. You will be prompted to open the solution in the current window, or open a new window.
+
+## Configure solution environment
+
+VS Code allows complete configurability of all aspects of the project.
+
+Locate `vcpkg-configuration.json` within the project. This file defines the components used.
+
+Right-click on this file, and select `Configure Arm Tools Environment` to open the configuration panel.
+
+From the various pull-downs, ensure that the most up to date versions of the following are selected:
+
+* Arm CMSIS-Toolbox
+* Arm Compiler for Embedded
+* Arm Debugger
+* Arm Virtual Hardware for Cortex-M based on Fast Models
+* Kitware's cmake tool
+* Ninja Build
+
+Others can be set as `None` as they are not needed for this example.
+
+If you open `vcpkg-configuration.json` in the text editor you will see these set as selected. Close the file to save.
+
+All necessary components will be downloaded and installed as necessary (if not already installed).
+
+## Configure CMSIS options
+
+Select `CMSIS` from the Extensions icon list. You will see the project structure.
+
+Move cursor over the top-level `Project`, and click on `Manage Software Components`. This is the view to add CMSIS Software Packs to your project.
+
+Select `CMSIS > Core` and `Device > Startup` (these are likely selected by default).
+
+Close this view to save.
+
+## Configure debug with the FVP
```

Comment:
Tried for VSCode on Mac, using the FVPs-on-MacOS, where it builds successfully. On running the Debugger: 
```
Failed to start model 'FVP_MPS2_Cortex-M4'. Please check that the model is installed correctly and its installation directory is included in your PATH environment variable
```
I verified that this was indeed added to the path, and runs fine when launched in the terminal.

Not sure if there's anything here that can be clarified, for either Mac or Windows.

Reply:
Cannot replicate, but expected if you were not able to configure project to use FVP library in previous steps.